.github/workflows/daily.yaml

```diff
@@ -52,7 +52,22 @@ jobs:
           VERSION_CODE: ${{ steps.rel_number.outputs.version-code }}
         run: ./gradlew bundleFullRelease
 
+      - name: Create cache file
+        run: |
+          mkdir check-SHA
+          echo ${{ github.sha }} > github-sha.txt
+
+      - name: Check SHA
+        id: check_sha
+        uses: actions/cache@v1
+        with:
+          path: check-SHA
+          key: check-SHA-${{ github.sha }}
+
       - name: Deploy to Playstore Beta
+        # Only run if a new commit is present.  This should prevent a new beta being created
```

Comment:
I noticed that when I update from firebase and there is no other new update for the day the play store still sends me the same build. Is that to be expected or is it technically a new build?

Reply:
They are built at different times but with the same version code.  They are essentially the same build, but android allows you to reinstall the APK so depending on who you install it from it might prompt to "update" but it won't do anything.